stumpy/stumpi.py

```diff
@@ -471,6 +453,61 @@ def _update(self, t):
         self._Σ_T = Σ_T_new
         self._T_subseq_isconstant = T_subseq_isconstant_new
 
+    @staticmethod
```

Comment:
@seanlaw 
Is this a good idea? I know if I move it to `core` (and write test for it), I do not need to define it again in `aampi`. But I wanted to limit the usability of this function to these classes only for now.

Reply:
To be consistent, I would move it to `core.py`